openhtf/util/validators.py

```diff
@@ -75,9 +76,30 @@ def create_validator(name, *args, **kwargs):
 
 _identity = lambda x: x
 
+
+class ValidatorBase(object):
```

Comment:
Take a look at my edit on line 27; it segues into the non-class example (which was already there). I think that would be sufficient, right?